src/core/dev/hw_queue_tx.cpp

```diff
@@ -2001,18 +2003,30 @@ bool hw_queue_tx::expand_doca_inventory()
 
 bool hw_queue_tx::expand_doca_task_pool(bool is_lso)
 {
+    if (m_task_list_count >= safe_mce_sys().tx_queue_max_elements) {
+        hwqtx_logfunc("silent packet drop, can't expand task pool!");
```

Comment:
Please use capital letters in the start of logging even they are logfunc. Also no need for marks such as '!'.